server/src/controllers/Events/resolver.ts

```diff
@@ -273,17 +273,62 @@ ${venue.postal_code} <br>
   }
 
   @Mutation(() => Boolean)
-  async sendEventInvite(@Arg('id', () => Int) id: number) {
+  async sendEventInvite(
+    @Arg('id', () => Int) id: number,
+    @Arg('emailGroups', () => [String], {
+      nullable: true,
+      defaultValue: ['interested'],
+    })
+    emailGroups: Array<string>,
+  ) {
     const event = await Event.findOne(id, {
-      relations: ['venue', 'chapter', 'chapter.users', 'chapter.users.user'],
+      relations: [
+        'venue',
+        'chapter',
+        'chapter.users',
+        'chapter.users.user',
+        'rsvps',
+        'rsvps.user',
+      ],
     });
 
     if (!event) throw new Error("Can't find event");
 
     // TODO: the default should probably be to bcc everyone.
-    const addresses = event.chapter.users
-      .filter((role) => role.interested)
-      .map(({ user }) => user.email);
+    const addresses: string[] = [];
+    if (emailGroups.includes('interested')) {
+      const interestedUsers: string[] = event.chapter.users
+        .filter((role) => role.interested)
+        .map(({ user }) => user.email);
+
+      addresses.push(...interestedUsers);
+    }
```

Comment:
```suggestion
```
`emailGroups.includes('interested')` is always false, so we can safety remove this block.

Reply:
@ojeytonwilliams , This **_interested_** option is to send emails when the new event is created. and I did not want to break the existing functionality, So I just kept it there.

Reply:
Yep, I understand.  The other `emailGroups` are handled correctly, though, so you've kept the functionality we want (namely, that we're not spamming users that are not _interested_).

We know, from the way you set up the modal, that `'interested'` will never be an element of `emailGroups`, though.

Reply:
So will we not be sending an email for the users who has user_chapter_role.interested=true, when the new Event is created?

Reply:
Nope, you're quite right, this approach makes sense.

What I will do is refactor this a little in another PR, so that the automatic invitation and the organiser triggered one are distinct mutations.  Anyways, that can wait :)